app/controllers/items_controller.rb

```diff
@@ -49,7 +49,7 @@ def create
       # with the error + the invalid parameters
       @item = current_organization.items.new(item_params)
 
-      flash[:error] = "Something didn't work quite right -- try again?"
+      flash[:error] = "An item with that name already exists (could be an inactive item)."
```

Comment:
@Gabe-Torres  The text change here is why the `item_system_spec` test is failing.

The failing test is looking for the text **_didn't work_** when it simulates creation of a new item with empty attributes. With the changed text that that is not happening anymore.

Failing test code:
```
  it "can create a new item with empty attributes as a user" do
    visit new_item_path
    click_button "Save"

    expect(page.find(".alert")).to have_content "didn't work"
  end
  ```
  
  I was thinking you could show the new error message when the uniqueness check fails and the old one when that does not happen.

Reply:
Gotcha! I just pushed up some changes that keep the old error message when it is not failing for uniqueness. Thank you for the feedback!